finagle-mysql/src/main/scala/com/twitter/finagle/mysql/CanBeParameter.scala

```diff
@@ -71,6 +71,29 @@ object CanBeParameter {
     }
   }
 
+  implicit val bigIntCanBeParameter = {
+    new CanBeParameter[BigInt] {
+      def sizeOf(param: BigInt) = 8
+      def typeCode(param: BigInt) = Type.LongLong
+      def write(writer: MysqlBufWriter, param: BigInt) = {
+        val byteArray: Array[Byte] = param.toByteArray
+        val lengthOfByteArray: Int = byteArray.length
+
+        if (lengthOfByteArray > 8) {
+          throw new BigIntTooLongException(size = lengthOfByteArray)
+        }
+
+        for (i <- 0 until lengthOfByteArray) {
+          writer.writeByte(byteArray(i))
```

Comment:
I think we need to reverse the order of the bytes we read out of `byteArray`: the `BigInt.toByteArray` method returns the bytes in big-endian order.